daemon/api_test.go

```diff
@@ -3965,3 +3976,219 @@ func (s *apiSuite) TestGetUserDetailsFromAssertionHappy(c *check.C) {
 	})
 	c.Check(err, check.IsNil)
 }
+
+func (s *apiSuite) TestPostCreateUserFromAssertion(c *check.C) {
```

Comment:
These tests all look similar, with small deltas. Would be nice to transform them into a table that is just the deltas, and run on a loop.

Obviously, not a priority for next week.